src/wizards/tokenSwapDealWizard/components/stageButtons/stageButtons.ts

```diff
@@ -1,13 +1,17 @@
 import { autoinject } from "aurelia-framework";
 import { bindable } from "aurelia-typed-observable-plugin";
-import { WizardService } from "../../../services/WizardService";
+import { IDealRegistrationTokenSwap } from "entities/DealRegistrationTokenSwap";
+import { IWizardState, WizardService } from "../../../services/WizardService";
 
 import "./stageButtons.scss";
 
 @autoinject
 export class stageButtons {
   @bindable wizardManager: any;
-  @bindable showSubmit = false;
+  @bindable showSubmit;
```

Comment:
Removed the default to better control when to show the submit button.
(Related change: line 28)